docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: antora-ui-default
```

Comment:
We should remove the entire `docs` directory.

Reply:
Removed